demos/README.md

```diff
@@ -17,12 +17,14 @@ This set of demos shows how real-world apps can be easily run inside SGX enclave
 * [fish](fish/): A demo of [FISH](https://fishshell.com) shell script.
 * [flink](flink/): A demo of [Apache Flink](https://flink.apache.org).
 * [font](font/font_support_for_java): A demo of supporting font with Java.
-* [https_server](https_server/): A HTTPS file server based on [Mongoose Embedded Web Server Library](https://github.com/cesanta/mongoose).
 * [grpc](grpc/): A client and server communicating through [gRPC](https://grpc.io).
+* [https_server](https_server/): A HTTPS file server based on [Mongoose Embedded Web Server Library](https://github.com/cesanta/mongoose).
 * [openvino](openvino/) A benchmark of [OpenVINO Inference Engine](https://docs.openvinotoolkit.org/2019_R3/_docs_IE_DG_inference_engine_intro.html).
 * [redis](redis/): A demo of [redis](https://redis.io).
+* [sofaboot](sofaboot/): A demo of [SOFABoot](https://github.com/sofastack/sofa-boot).
```

Comment:
I think it would be helpful to give a brief description about SOFABoot since the project is not that well known.
```
[sofaboot](sofaboot/): A demo of [SOFABoot](https://github.com/sofastack/sofa-boot), an open source Java development framework based on Spring Boot.
```

Reply:
Done